assets/sass/vendor/_mdc-checkbox.scss

```diff
@@ -31,4 +31,8 @@
 		width: 100%;
 		z-index: 1;
 	}
+
+	.spinner {
+		margin: 0;
+	}
```

Comment:
I realize this was in the IB but let's apply this using an inline style (which are already being used within `Spinner` otherwise I'd opt for a utility class here). We can just merge the given `style` object with the inner one.

This stylesheet is more for calibrating the material styles for SK and most exceptions that we might need for it should be doable with utilities where needed 👍

Reply:
I think that's the method I originally wrote in the IB but I changed it after this feedback from @eugene-manuilov: https://github.com/google/site-kit-wp/issues/3304#issuecomment-920241907

CCing him in case he had any specific concerns.

I'll switch it to use the `style` prop for now 👍

Reply:
We generally don't use inline styles in components although there are a few instances left. Rather than unpacking the existing inline styles on `Spinner` here I think it's okay to extend them since they'd take precedence over what would be defined in a stylesheet. I'll take a note to refactor the `Spinner` later 👍